common/src/main/java/org/apache/sedona/common/utils/RasterUtils.java

```diff
@@ -192,6 +192,10 @@ public static GridCoverage2D clone(RenderedImage image, GridGeometry2D gridGeome
      * @return A new GridCoverage2D object.
      */
     public static GridCoverage2D create(WritableRaster raster, GridGeometry2D gridGeometry, GridSampleDimension[] bands, Double noDataValue) {
+      return create(raster, gridGeometry, bands, noDataValue, null);
```

Comment:
Why make changes to the `create` function? How will this affect other functions?

Reply:
I want to extend it to be able to create a raster with a given map of properties. This will not affect other functions since I am reusing this function with a null argument for the properties which was being done originally.